lib/alchemy/host.rb

```diff
@@ -0,0 +1,5 @@
+module Alchemy
+  mattr_accessor :enable_subdomain_locale
+
+  @@enable_subdomain_locale = false
+end
```

Comment:
Since it's a config, shouldn't this be in https://github.com/nimbl3/alchemy_cms/blob/development/lib/alchemy/config.rb or https://github.com/nimbl3/alchemy_cms/blob/development/lib/alchemy/configuration_methods.rb?

Reply:
@olivierobert I move this configuration to [config.rb](https://github.com/nimbl3/alchemy_cms/blob/development/lib/alchemy/config.rb) and we can override that from `Alchemy.enable_subdomain_locale = true` to `Alchemy::Config.enable_subdomain_locale = true`